Makefile

```diff
@@ -13,8 +13,12 @@
 
 GO     ?= GO15VENDOREXPERIMENT=1 go
 GOPATH := $(firstword $(subst :, ,$(GOPATH)))
-PROMU  ?= $(GOPATH)/bin/promu
-pkgs    = $(shell $(GO) list ./... | grep -v /vendor/)
+GOOS   := $(shell uname -s | tr A-Z a-z)
+GOARCH := $(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m)))
```

Comment:
The circleci build's error is coming from those two lines. You are forcing `GOOS` and `GOARCH` with the `:=` operator, when `golang-builder` docker images are trying redefining those vars when crossbuilding.

Please remove those lines, and modify `$(GOPATH)/bin/promu promu` and `$(GOPATH)/bin/staticcheck` targets based on my comment below.